examples/libs/aptos/feepayer-tx/main.ts

```diff
@@ -0,0 +1,116 @@
+import { DfnsApiClient } from '@dfns/sdk'
+import { AsymmetricKeySigner } from '@dfns/sdk-keysigner'
+
+import * as dotenv from 'dotenv'
+
+dotenv.config()
+
+const initDfnsWallet = async (walletId: string) => {
+  const signer = new AsymmetricKeySigner({
+    credId: process.env.DFNS_CRED_ID!,
+    privateKey: process.env.DFNS_PRIVATE_KEY!,
+  })
+
+  const dfnsClient = new DfnsApiClient({
+    appId: process.env.DFNS_APP_ID!,
+    authToken: process.env.DFNS_AUTH_TOKEN!,
+    baseUrl: process.env.DFNS_API_URL!,
+    signer,
+  })
+
+  return DfnsWallet.init({
+    walletId,
+    dfnsClient,
+  })
+}
+
+async function main() {
+  const sender = await initDfnsWallet(process.env.APTOS_SENDER_WALLET_ID!)
+  const feePayer = await initDfnsWallet(process.env.APTOS_FEE_PAYER_WALLET_ID!)
+
+  // If you want to provide your own fullnode/indexer URL, you need to use `Network.CUSTOM`
+  const aptosConfig = new AptosConfig({
+    network: Network.TESTNET,
+  })
+  const client = new Aptos(aptosConfig)
+
+  console.log('sender address: ', sender.address)
+  console.log('feepayer address: ', feePayer.address)
+
+  const [balance, feePayerBalance] = await Promise.all([
+    client.account.getAccountAPTAmount({accountAddress: sender.address}),
+    client.account.getAccountAPTAmount({accountAddress: feePayer.address}),
+  ])
+  console.log(`initial sender balance: ${balance}, feepayer balance: ${feePayerBalance}`, )
+
+  const tx = await client.transaction.build.simple({
+    sender: sender.address,
+    data: {
+      function: '0x1::coin::transfer',
+      typeArguments: [APTOS_COIN],
+      // receiver + amount
+      functionArguments: ["0x5bdc24cb9033286ffe19f436145b9e2267dd03b0fd0d422459d381a6431d39ba", '1'],
+    },
+    withFeePayer: true,
+  })
+
+  console.log(`building fee payer transaction`)
+
+  // We build a SignedTransaction to sign. Since we are doing a feepayer transaction, we need
+  // to build a TransactionAuthenticatorFeePayer to indicate DFNS wallet that we want to sign a
+  // transaction with a given fee payer
+  const txWithAuth = new SignedTransaction(
+    tx.rawTransaction,
+    new TransactionAuthenticatorFeePayer(
+      // sender
+      new AccountAuthenticatorEd25519(
+        new Ed25519PublicKey(sender.publicKey),
+        // mock signature
+        new Ed25519Signature('0'.repeat(128))
+      ),
+      [],
+      [],
+      // fee payer
+      {
+        address: new AccountAddress(hexToBuffer(feePayer.address)),
+        // mock authenticator. it will be replaced when the fee payer will sign
+        authenticator: new AccountAuthenticatorEd25519(
+          new Ed25519PublicKey('0'.repeat(64)),
+          new Ed25519Signature('0'.repeat(128))
+        ),
+      }
+    )
+  )
+
+  // sender sign
+  let signed = (await sender.signTransaction(txWithAuth))
+  console.log(`sender signed the transaction`)
+
+  // feePayer sign
+  signed = (await feePayer.signTransaction(signed))
+  console.log(`fee payer signed the transaction`)
```

Comment:
would like to simplify this down to

```suggestion
  const tx = await client.transaction.build.simple({
    sender: sender.address,
    data: {
      function: '0x1::coin::transfer',
      typeArguments: [APTOS_COIN],
      // receiver + amount
      functionArguments: ["0x5bdc24cb9033286ffe19f436145b9e2267dd03b0fd0d422459d381a6431d39ba", '1'],
    },
    withFeePayer: true,
  })
  
  tx.feePayerAddress = feePayer.address

  // sender sign
  const partial = await sender.signTransaction(tx)
  console.log(`sender signed the transaction`)

  // feePayer sign
  const signed = await feePayer.signTransaction(partial)
  console.log(`fee payer signed the transaction`)
```